### PR TITLE
Docs: Add getting started draft

### PR DESCRIPTION
**Pull request by videlanicolas**
### Checklist for this pull request

- [x] I have read the [guidelines for contributing](../CONTRIBUTING.md).
- [x] I have checked that my code additions did not fail tests.

### Description

#41 

This first draft outlines what we expect from the "Getting Started" document. It should tell what is Inkgd and easily guide users how to install this addon. Furthermore it'll point to the next section where it'll show a step by step tutorial on how to use this addon on a sample game.

This first draft is missing hyperlinks and images, but given that this is a draft and we want to put the information out there in the documentation to see how it looks, I think it's good enough for a commit. Further improvements will see adding images to guide users.

### CODE REVIEW

File: docs/source/getting_started/index.rst [hunk 1]
```diff
@@ -4,4 +4,49 @@
...
+
+Installing the addon
+====================
+
+You have two ways of installing this addon: the easy and convenient way or the slightly more difficult way.
+
+The easy way
+************
+
+Open a new Godot project, then navigate to "AssetLib" on the top of the screen, search for "inkgd". Select "inkgd" from author "ephread", in the popup screen click "Download". Once the addond is downloaded to your computer a popup screen will show up, this describes the new files it'll add to your project. By default it should select everything, if this is not the case then select all files, click "Install". After the installation is done a popup message will show saying the addon was added to your project, click "Ok" to close the popup.
```
Comment (ephread) [inline docs/source/getting_started/index.rst:26]: Once the _addond_ […] → Once the _addon_ […]

File: docs/source/getting_started/index.rst [hunk 2]
```diff
@@ -4,4 +4,49 @@
...
-TODO: Do this section.
\ No newline at end of file
+This article will help you figure out how to start using Ink with GDScript in Godot.
+
+What is Inkgd?
+==============
+
+Inkgd is Ink for Godot, more precisely GDScript. It's a GDScript API that you import as an addon in your Godot project that can read an Ink story (file with .ink.json suffix) and execute Ink's methods directly in GDScript. With Inkgd you'll be able to load your story and get the text from each knot, choice and diverts, then use the text as-is. It also binds variable changes and external functions with custom functions found in your Godot scripts.
+
+A normal use of Inkgd would be to load a compiled story in '.ink.json' and then call "continue_story" until a branch is found. Here the choices are read as an array of strings and the story waits for you to input a choice. Once you submit the selected choice (represented by the index of the choices array) to the story, you repeat the process (i.e. keep calling "continue_story" until a branch is found).
```
Comment (ephread) [inline docs/source/getting_started/index.rst:14]: I feel an inline ` ``continue_story`` ` would look slightly better than `"continue_story"`, what do you think? (Unless that's not supported)

**Review by ephread — CHANGES_REQUESTED** [review]
This is looking really good! Thanks again for taking the time to improve the documentation.

I'm nitpicking a bit, but it's great otherwise.

File: docs/source/getting_started/index.rst [hunk 3]
```diff
@@ -4,4 +4,49 @@
 Introduction
 ============
 
-TODO: Do this section.
\ No newline at end of file
+This article will help you figure out how to start using Ink with GDScript in Godot.
+
+What is Inkgd?
```
[the remark this answers is not in the document]
Comment (videlanicolas) [reply docs/source/getting_started/index.rst:9]: Sounds good, I'll change this.

File: docs/source/getting_started/index.rst (hunk 1 above)
Comment (videlanicolas) [reply docs/source/getting_started/index.rst]: Done.

File: docs/source/getting_started/index.rst (hunk 2 above)
Comment (videlanicolas) [reply docs/source/getting_started/index.rst]: Yeah using backticks here sounds alright, it's the name of the method as well so I think is ok.

**Comment by ephread** [comment]
LGTM!

**Review by ephread — APPROVED** [review]

**Pull request merged**